pkg/virtctl/credentials/password/password.go

```diff
@@ -88,9 +88,9 @@ func runSetPasswordCommand(clientConfig clientcmd.ClientConfig, cmdFlags *passwo
 	}
 
 	if !cmdFlags.Force {
-		secret, err := cli.CoreV1().Secrets(vm.Namespace).Get(cmd.Context(), secretName, metav1.GetOptions{})
-		if err != nil {
-			return fmt.Errorf("error getting secret \"%s\": %w", secretName, err)
+		secret, err2 := cli.CoreV1().Secrets(vm.Namespace).Get(cmd.Context(), secretName, metav1.GetOptions{})
```

Comment:
Please use a more meaningful name for the error, like `errs` or `errSecret`.

Reply:
done